### PR TITLE
Added pypi-switch config and docs

### PR DESCRIPTION
**Pull request by tropxy**
Due to the change in the OCPP repo where we want to add the mqtt_api as dependency, we also need to provide the credentials for the repo

### CODE REVIEW

File: base.dockerfile [hunk 1]
```diff
@@ -11,8 +15,15 @@ ENV POETRY_HOME="/opt/poetry" \
...
+
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3
 
+
 WORKDIR $PYSETUP_PATH
 COPY pyproject.toml poetry.lock ./
-RUN poetry install --no-dev
\ No newline at end of file
+RUN poetry config http-basic.pypi-switch $PYPI_USER $PYPI_PASS
+RUN poetry install -vvv --no-interaction --no-dev
```
Comment (HugoJP1) [inline base.dockerfile:29]: minor - you can remove the -vvv now, otherwise that will spam the logs

Comment (HugoJP1) [inline base.dockerfile:29]:
```suggestion
RUN poetry install --no-interaction --no-dev
```

Comment (tropxy) [reply base.dockerfile:29]: damn it...I removed it when I opened another PR but then something weird happened with it and the tests were not passing and I opened a new PR and forgot to remove that flag....thanks!

Comment (tropxy) [reply base.dockerfile]: done

**Review by HugoJP1 — APPROVED** [review]

**Review by proelke — APPROVED** [review]

**Pull request merged**